src/test/kotlin/no/skatteetaten/aurora/boober/unit/S3StorageGridProvisionerTest.kt

```diff
@@ -0,0 +1,79 @@
+package no.skatteetaten.aurora.boober.unit
+
+import assertk.assertThat
+import assertk.assertions.isEqualTo
+import com.fasterxml.jackson.module.kotlin.jacksonObjectMapper
+import io.mockk.every
+import io.mockk.mockk
+import no.skatteetaten.aurora.boober.feature.OperationScopeFeature
+import no.skatteetaten.aurora.boober.service.HerkimerService
+import no.skatteetaten.aurora.boober.service.ResourceClaimHerkimer
+import no.skatteetaten.aurora.boober.service.ResourceHerkimer
+import no.skatteetaten.aurora.boober.service.ResourceKind.StorageGridObjectArea
+import no.skatteetaten.aurora.boober.service.resourceprovisioning.S3StorageGridProvisioner
+import no.skatteetaten.aurora.boober.service.resourceprovisioning.SgProvisioningRequest
+import no.skatteetaten.aurora.boober.service.resourceprovisioning.StorageGridCredentials
+import org.junit.jupiter.api.Test
+import java.time.LocalDateTime.now
+import kotlin.random.Random
+
+class S3StorageGridProvisionerTest {
```

Comment:
Jeg klarer ikke helt å se på stående fot om S3StorageGridProvisioner er dekket noe annet sted. Er den dekket av noen integrasjonstester? 
Blir feilhåndtering testet osv...

Reply:
Happyday blir dekket av en integrasjonstest, men det er noen feilscenarier som ikke er dekket. Vi bør ta en liten prat om testing i boober, fordi det er ganske drøyt hva som må til for å få på plass endel av disse testene. Lager en oppgave på å forbedre dette.

Reply:
I see. Ja er enig at f.eks DeployFacadeTest er massiv og inneholder masse.